tools/test.py

```diff
@@ -76,7 +76,7 @@ def main(argv):
 
     test_cc = os.path.join(build_dir, "test_cc" + executable_suffix)
     check_exists(test_cc)
-    run([test_cc])
+    run([test_cc, "--expose-gc"])
```

Comment:
Can you move the `--expose-gc` into C++ so that it is always specified. That should be done here:
https://github.com/denoland/deno/blob/635ad2a18ecd26690e611f54062ae899fe307f96/core/libdeno/test.cc#L29

(It's slightly tricky, I guess you have to allocate a new argv with size argc+1 and copy over the existing flags, and add the new "--expose-gc")